docs_sources/examples.md

```diff
@@ -139,3 +147,10 @@ results = exported_pipeline.predict(testing_features)
 ## Titanic survival analysis
 
 To see the TPOT applied the Titanic Kaggle dataset, see the Jupyter notebook [here](https://github.com/rhiever/tpot/blob/master/tutorials/Titanic_Kaggle.ipynb). This example shows how to take a messy dataset and preprocess it such that it can be used in scikit-learn and TPOT.
+
+## Portuguese Bank Marketing
+
+The corresponding Jupyter notebook, containing the associated data preprocessing and analysis, can be found [here](https://github.com/rhiever/tpot/blob/master/tutorials/Portuguese%20Bank%20Marketing/Portuguese%20Bank%20Marketing%20Stratergy.ipynb).
+
+## MAGIC Gamma Telescope
+The corresponding Jupyter notebook, containing the associated data preprocessing and analysis, can be found [here](https://github.com/rhiever/tpot/blob/master/tutorials/MAGIC%20Gamma%20Telescope/MAGIC%20Gamma%20Telescope.ipynb).
```

Comment:
Thank you for catching the out-of-date jupyther notebook. Please update/replace this .ipynb file for updating `from tpot import TPOT` to `from tpot import TPOTClassifier`